shuttle/db/db_functions.py

```diff
@@ -102,3 +104,56 @@ def username_search(username):
             conn.close()
         # if mybethel can't get the data, then prevent anything from loading
         return abort(503)
+
+
+def commit_schedule_to_db(table, all_locations):
+    sql = "DELETE FROM SHUTTLE_SCHEDULE"
+    query(sql, 'write')
+    all_locations = [i.upper() for i in all_locations]
+    for i in range(len(table)):
+        location = ""
+        for j in range(len(table[i])):
+            arrival_time = 0
+            if table[i][j].upper() in all_locations:
+                location = table[i][j].upper()
+            elif table[i][j] == '-' or table[i][j] == 'DROP':
+                arrival_time = '01-AUG-00 01.00.00.000000000 AM'
+            elif table[i][j].split(':') is not None:
+                split_time = table[i][j].split(':')
+                if int(split_time[0]) == 12 or 1 <= int(split_time[0]) < 6:
+                    joined_time = '.'.join(split_time)
+                    arrival_time = '01-SEP-00 ' + joined_time + '.00.000000000 PM'
+                else:
+                    joined_time = '.'.join(split_time)
+                    arrival_time = '01-SEP-00 ' + joined_time + '.00.000000000 AM'
+            else:
+                return "data in calendar does not match specified format"
+            if j is not 0:
+                sql = "INSERT INTO SHUTTLE_SCHEDULE (LOCATION,ARRIVAL_TIME) VALUES (" + "\'" + location + \
+                      "\',\'" + arrival_time + "\')"
+                query(sql, 'write')
+    return "The Schedule has been submitted"
+
+
+def commit_shuttle_request_to_db(location):
```

Comment:
You could get rid of `_to_db` and have it just be `commit_shuttle_request` since it is in the db_functions.py file so it is inferred that it is to the db.